nncf/torch/quantization/layers.py

```diff
@@ -744,12 +768,15 @@ def signed(self, signed: bool):
         self.set_levels()
 
     def quantize(self, x, execute_traced_op_as_identity: bool = False):
+        with DisableTorchFunction():
```

Comment:
@AlexanderDokuchaev please pay attention to this.
Otherwise,  tests from tests/torch2/function_hook/quantization/test_quantized_graphs.py fail:
https://github.com/openvinotoolkit/nncf/actions/runs/13720847384/job/38375801335?pr=3322
![Uploading image.png…]()

Reply:
noted, i will add some operations to filter